django/test/testcases.py

```diff
@@ -1345,3 +1351,26 @@ def setUpClass(cls):
     def tearDownClass(cls):
         super().tearDownClass()
         cls._lockfile.close()
+
+
+class WidgetTestCase(SimpleTestCase):
+    """
+    For testing widget rendering in Django's test suite (not a public API).
+    """
```

Comment:
It should be good as long there's no foreign tests or models that get imported along the way. From what I can see it should be good!